src/app/mesh/qgsrenderermeshpropertieswidget.cpp

```diff
@@ -168,7 +168,6 @@ void QgsRendererMeshPropertiesWidget::onActiveScalarGroupChanged( int groupIndex
   mMeshRendererScalarSettingsWidget->setActiveDatasetGroup( groupIndex );
   mMeshRendererScalarSettingsWidget->syncToLayer();
   mContoursGroupBox->setChecked( groupIndex >= 0 );
-  mContoursGroupBox->setEnabled( groupIndex >= 0 );
```

Comment:
if `groupIndex >= 0` it means that the user does not selected any group in the group tree widget. So I do not understand what it help if the checkbox for enabling/disabling is available. Or what it does when user enables the vector renderer for groupIndex == -1?

Reply:
True... I have made this PR because I found inconsistent that vector/contour can be disabled from those checkbox, but could not be enable in some cases.
But since disable/enable contour or vector from the group tree widget is fully functional, maybe those check boxes were not necessary anymore?

Reply:
I think it is fine as it is.... User can disable it now from both places ...